cpp/src/flightsql_odbc/amazon-timestream-odbc-driver/flightsql-readme.md

```diff
@@ -0,0 +1,29 @@
+# FligthSQL ODBC Driver
```

Comment:
FYI - after your PR is merged, I can move this read me file as part of my PR #9. Or if #9 is merged first, I might have to ask you to please move this file to `cpp/src/flightsql_odbc/flightsql-odbc`

Reply:
#9  is merged, please rebase your branch onto `GAC-odbc-driver` and move the readme to `cpp/src/flightsql_odbc/flightsql-odbc`. 
Tip: On GitHub Desktop, you can drag 2 commits together to form a squash:
https://docs.github.com/en/desktop/managing-commits/squashing-commits-in-github-desktop

![image](https://github.com/Bit-Quill/arrow/assets/96995091/aab99d8a-d357-486f-a7aa-9fde79918052)

Reply:
Done